{"name": "test_login_page", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nPlease use the Async API instead.", "trace": "def test_login_page() :\n>       with sync_playwright() as playwright :\n\nbasics\\RahulShetty_Academy_Login_Page_Test.py:4: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <playwright.sync_api._context_manager.PlaywrightContextManager object at 0x000001661EFC6B40>\n\n        def __enter__(self) -> SyncPlaywright:\n            try:\n                self._loop = asyncio.get_running_loop()\n            except RuntimeError:\n                self._loop = asyncio.new_event_loop()\n                self._own_loop = True\n            if self._loop.is_running():\n>               raise Error(\n                    \"\"\"It looks like you are using Playwright Sync API inside the asyncio loop.\n    Please use the Async API instead.\"\"\"\n                )\nE               playwright._impl._errors.Error: It looks like you are using Playwright Sync API inside the asyncio loop.\nE               Please use the Async API instead.\n\n.venv\\Lib\\site-packages\\playwright\\sync_api\\_context_manager.py:47: Error"}, "start": 1728539958973, "stop": 1728539958973, "uuid": "b4635677-b678-40e3-87be-1b154cde7c34", "historyId": "f89ad94655f75dd6024ccae817c5fa6d", "testCaseId": "f89ad94655f75dd6024ccae817c5fa6d", "fullName": "basics.RahulShetty_Academy_Login_Page_Test#test_login_page", "labels": [{"name": "parentSuite", "value": "basics"}, {"name": "suite", "value": "RahulShetty_Academy_Login_Page_Test"}, {"name": "host", "value": "ALIPLR8365"}, {"name": "thread", "value": "22776-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "basics.RahulShetty_Academy_Login_Page_Test"}]}